### PR TITLE
[web3t] Remove incorrect usages of 'seeder'

### PR DESCRIPTION
**Pull request by geoknee**
These mostly arise out of the (tempting but incorrect) assumption that anyone who uploads is a seeder.

### CODE REVIEW

**Comment by geoknee** [comment]
@spersico we currently have 

https://github.com/statechannels/monorepo/blob/5ece4ea6918d15ab60e6fa309c6aef1980b9fe5e/packages/web3torrent/src/types.ts#L3-L13

Which doesn't do as much justice as I would like to this issue. 

Should we have another status such as `Downloading&Uploading` ? Or rename `Downloading`?  And add this to to `UploadingStatuses`?

**Comment by spersico** [comment]
@geoknee Ummm I think Downloading it's a valid state (for example when there are only one seeder and only one leecher, the leecher will only download). 

Also, that status was UI related, mostly (that's why it's not in the library, but on the client, and it's maintained by the torrent-status-checker utils). 

I like the idea of adding a status of Downloading&Uploading and add that to UploadingStatuses.

And we should stop showing one OR the other, and allow for **both sections** to be shown at the same time: https://github.com/statechannels/monorepo/blob/7343f348409fdb7717de2f2e0bcb9d4b69013493/packages/web3torrent/src/components/torrent-info/TorrentInfo.tsx#L40-L51

**Comment by geoknee** [comment]
@spersico Sounds good, I can address this in a future PR.

The current state of the UI is to *always* show `UploadInfo`, and show `DownloadInfo` conditional on the download status. The plan would be to independently show both, but each has its own condition on the status. And also to introduce a `Downloading&Uploading` status that will activate both components.

File: packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {WebTorrentContext} from '../../../clients/web3torrent-client';
 export type UploadInfoProps = {
   wires: PaidStreamingWire[];
   channels: Dictionary<ChannelState>;
-  pseType: 'seeder' | 'leecher';
+  participantType: 'payer' | 'beneficiary';
```
Comment (tomclose) [inline packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx]: Wondering whether `data-sender` and `data-receiver` would be even clearer?

Comment (geoknee) [reply packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx]: This component is about Channels primarily, though. And these are the terms we are using in the `PaymentChannelClient`.

**Pull request merged**